test/e2e/network/netpol/model.go

```diff
@@ -325,4 +355,7 @@ func (c *Container) Spec() v1.Container {
 			},
```

Comment:
@rajaskakodkar you can use `Resources` as a direct field, and bring the comment up.

Reply:
added this in the recent commit